### PR TITLE
Implements the login workflow.

### PR DESCRIPTION
**Pull request by JuliusAlexanderIV**
Sorry it's so big.

### CODE REVIEW

File: app/src/main/java/com/whereuat/whereu/HttpRequestHandler.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                conn.setConnectTimeout(CONNECT_TIMEOUT);
+                conn.setRequestMethod("POST");
+                conn.setRequestProperty("Content-Type", "application/json");
+                conn.setDoInput(false);
+                conn.setDoOutput(false);
+                conn.connect();
+
+                DataOutputStream os = new DataOutputStream(conn.getOutputStream());
+                os.writeBytes(json_str);
+                os.flush();
```
Comment (JuliusAlexanderIV) [inline app/src/main/java/com/whereuat/whereu/HttpRequestHandler.java]: Remove

File: app/src/main/java/com/whereuat/whereu/HttpRequestHandler.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            try {
+                URL url = new URL(Constants.WHEREUAT_URL + route);
+                conn = (HttpURLConnection) url.openConnection();
+                conn.setReadTimeout(READ_TIMEOUT);
+                conn.setConnectTimeout(CONNECT_TIMEOUT);
+                conn.setRequestMethod("POST");
+                conn.setRequestProperty("Content-Type", "application/json");
+                conn.setDoInput(false);
+                conn.setDoOutput(false);
+                conn.connect();
```
Comment (kangp3) [inline app/src/main/java/com/whereuat/whereu/HttpRequestHandler.java]: I commented out 
```conn.setDoInput(false);
conn.setDoOutput(false);
conn.connect();```
and it still works. What gives

File: app/src/main/java/com/whereuat/whereu/HttpRequestHandler.java (hunk 1 above)
Comment (kangp3) [inline app/src/main/java/com/whereuat/whereu/HttpRequestHandler.java]: Commented out this too. Who knows.

**Comment by kangp3** [comment]
LGTM.
closes #12

**Pull request merged**